share/completions/apt.fish

```diff
@@ -91,6 +101,16 @@ __fish_apt_subcommand changelog -r -d 'Download and display package changelog'
 # Autoremove
 __fish_apt_subcommand autoremove -d 'Remove packages no longer needed as dependencies'
 
+# Autoremove
+__fish_apt_subcommand autopurge -d 'Remove packages no longer needed as dependencies and delete their config files'
+
+# Clean
+__fish_apt_subcommand clean -d 'Clean all cache without lock'
+
+# Autoclean
+__fish_apt_subcommand autoclean -d 'Clean cache largely useless'
+#__fish_apt_subcommand auto-clean -d 'Clean cache largely useless'
```

Comment:
This doesn't add the synonym, but `disk-upgrade` is there.

We should either add both or neither. To be honest my preference is for neither, because they just clog the suggestions.

Reply:
Maybe better to change suggestion to like `same as full-update`. 

In my opinion, to skip completion of `auto-clean` is OK because this alias is not really common (`bash` do not complete for it ) and has same beginning with `autoclean`.

As for `disk-upgrade` the problem is different. Though `disk-upgrade` is out-of-date, because it is used for ages for `apt-get`, many `apt` users are still more familiar to it rather than `full-upgrade`, so that to skip this completion can cause confusion.

Reply:
Hold on, did you mean `dist-upgrade`? With a "t" in "dist" instead of "disk"? Short for "distro"?

I know that one, I've never heard of "disk-upgrade".

Reply:
> Hold on, did you mean `dist-upgrade`? With a "t" in "dist" instead of "disk"? Short for "distro"?
> 
> I know that one, I've never heard of "disk-upgrade".

Sorry, I type mistake here. (Oh I am too often to spell wrong) . You are right.

Thanks.